app/src/androidTest/java/org/mozilla/fenix/ui/ThreeDotMenuMainTest.kt

```diff
@@ -54,9 +62,25 @@ class ThreeDotMenuMainTest {
             verifyDesktopSite()
             verifyWhatsNewButton()
             verifyHelpButton()
+            verifyCustomizeHomeButton()
             verifySettingsButton()
         }.openSettings {
             verifySettingsView()
+        }.openLanguageSubMenu {
+            // The app locale is null on Firebase
+            // We'll need to change the app language for the "Pocket" preference to be displayed
+            localeListIdlingResource =
+                RecyclerViewIdlingResource(
+                    activityTestRule.activity.findViewById(R.id.locale_list),
+                    2
+                )
+            IdlingRegistry.getInstance().register(localeListIdlingResource)
+            selectLanguage("English (Canada)")
```

Comment:
I don't know enough of the problem here but this seems really hacky. What do you mean by app locale is null? We're using Pixel 2, API 28 devices, what seems to be the problem?

>             // We'll need to change the app language for the "Pocket" preference to be displayed

That sounds like a bug in in our app, can you double check with the implementer?

If we need to change something in the OS settings, we should do it as a pre launch step, like changing locale in the https://github.com/mozilla-mobile/fenix/blob/main/app/src/androidTest/java/org/mozilla/fenix/helpers/HomeActivityTestRule.kt with a new function or something. Does that make sense?

Reply:
Not applicable anymore, please see my comment.